packages/blade/src/components/StepGroup/tokens.ts

```diff
@@ -0,0 +1,75 @@
+import type { StepGroupProps } from './types';
+import { size as sizeTokens } from '~tokens/global';
+
+const getLineSpacings = (
```

Comment:
Add a comment on what this is doing? Line spacing of what?

Reply:
spacings of line 🙈 let me add comment